spec/rails_helper.rb

```diff
@@ -1,3 +1,18 @@
+if ENV['RAILS_ENV'] != 'test'
```

Comment:
Trying not to clobber the dev environment when the test suite is run.

Reply:
Could maybe clean this up a bit? Maybe something like:


```ruby
ENV['RAILS_ENV'] = 'test'
if ENV['SOLR_URL'] && ENV['SOLR_TEST_URL']
   ENV['SOLR_URL'] = ENV['SOLR_TEST_URL']
 else
    raise "You will likely wipe out your Solr instance!\n"\
            "Please set SOLR_TEST_URL and try again."
end
```

Reply:
Yeah, could be cleaned up. The main problem with what you have there is that when neither `SOLR_URL` and `SOLR_TEST_URL` are set, your snippet  flags it as a problem, but in reality the defaults are fine:

https://github.com/ualbertalib/avalon/pull/283/files#diff-5ac8c463ec0cd8dfe961ec6f695f6d3aR5

I'll push something that hopefully illustrates the logic better.

Reply:
Ah good call, sounds good :+1: